FaustLive does not compile

**Describe the bug**

```
faustlive> ../../src/Network/FLServerHttp.cpp: In member function 'bool FLServerHttp::start()':
faustlive> ../../src/Network/FLServerHttp.cpp:78:32: error: invalid conversion from 'int (*)(void*, MHD_Connection*, const char*, const char*, const char*, const char*, size_t*, void**)' {aka 'int (*)(void*, MHD_Connection*, const char*, const char*, const char*, const char*, long unsigned int*, void**)'} to 'MHD_AccessHandlerCallback' {aka 'MHD_Result (*)(void*, MHD_Connection*, const char*, const char*, const char*, const char*, long unsigned int*, void**)'} [-fpermissive]
faustlive> 78 |                                &answerToConnection,
faustlive> |                                ^~~~~~~~~~~~~~~~~~~
faustlive> |                                |
faustlive> |                                int (*)(void*, MHD_Connection*, const char*, const char*, const char*, const char*, size_t*, void**) {aka int (*)(void*, MHD_Connection*, const char*, const char*, const char*, const char*, long unsigned int*, void**)}
faustlive> In file included from ../../src/Network/FLServerHttp.h:40,
faustlive> from ../../src/Network/FLServerHttp.cpp:22:
faustlive> /nix/store/xfjjhyxbrv9zylph4l6l7hpj4pys66k5-libmicrohttpd-0.9.71-dev/include/microhttpd.h:2428:45: note:   initializing argument 5 of 'MHD_Daemon* MHD_start_daemon(unsigned int, uint16_t, MHD_AcceptPolicyCallback, void*, MHD_AccessHandlerCallback, void*, ...)'
faustlive> 2428 |                   MHD_AccessHandlerCallback dh, void *dh_cls,
faustlive> |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~^~
faustlive> ../../src/Network/FLServerHttp.cpp: In static member function 'static int FLServerHttp::answerToConnection(void*, MHD_Connection*, const char*, const char*, const char*, const char*, size_t*, void**)':
faustlive> ../../src/Network/FLServerHttp.cpp:214:121: error: invalid conversion from 'int (*)(void*, MHD_ValueKind, const char*, const char*, const char*, const char*, const char*, uint64_t, size_t)' {aka 'int (*)(void*, MHD_ValueKind, const char*, const char*, const char*, const char*, const char*, long unsigned int, long unsigned int)'} to 'MHD_PostDataIterator' {aka 'MHD_Result (*)(void*, MHD_ValueKind, const char*, const char*, const char*, const char*, const char*, long unsigned int, long unsigned int)'} [-fpermissive]
faustlive> 214 |             con_info->postprocessor = MHD_create_post_processor(connection, POSTBUFFERSIZE, iteratePost, (void*)con_info);
faustlive> |                                                                                                                         ^
faustlive> |                                                                                                                         |
faustlive> |                                                                                                                         int (*)(void*, MHD_ValueKind, const char*, const char*, const char*, const char*, const char*, uint64_t, size_t) {aka int (*)(void*, MHD_ValueKind, const char*, const char*, const char*, const char*, const char*, long unsigned int, long unsigned int)}
faustlive> In file included from ../../src/Network/FLServerHttp.h:40,
faustlive> from ../../src/Network/FLServerHttp.cpp:22:
faustlive> /nix/store/xfjjhyxbrv9zylph4l6l7hpj4pys66k5-libmicrohttpd-0.9.71-dev/include/microhttpd.h:3465:49: note:   initializing argument 3 of 'MHD_PostProcessor* MHD_create_post_processor(MHD_Connection*, size_t, MHD_PostDataIterator, void*)'
faustlive> 3465 |                            MHD_PostDataIterator iter, void *iter_cls);
faustlive> |                            ~~~~~~~~~~~~~~~~~~~~~^~~~
faustlive> make[2]: *** [Makefile:497: tmp/FLServerHttp.o] Error 1
faustlive> make[2]: *** Waiting for unfinished jobs....
faustlive> make[2]: Leaving directory '/build/source/Build/fldir'
faustlive> make[1]: *** [Makefile:37: unix] Error 2
faustlive> make[1]: Leaving directory '/build/source/Build'
faustlive> make: *** [Makefile:30: all] Error 2
```


**To Reproduce**
1. `nix profile install self#faustlive -L`

**Notify maintainers**

@NegaNexus @magnetophon

--- a/pkgs/applications/audio/faust/faustlive.nix
+++ b/pkgs/applications/audio/faust/faustlive.nix
@@ -5,12 +5,12 @@
 
 stdenv.mkDerivation rec {
   pname = "faustlive";
-  version = "2.5.4";
+  version = "unstable-dev-2020-08-03";
   src = fetchFromGitHub {
     owner = "grame-cncm";
     repo = "faustlive";
-    rev = version;
-    sha256 = "0npn8fvq8iafyamq4wrj1k1bmk4xd0my2sp3gi5jdjfx6hc1sm3n";
+    rev = "c16565dc1b616ac0aad7c303c1997fa9e57177ab";
+    sha256 = "1ys661lp1xwz21vy12kwkg248jvjq1z9w433knkh0ldyy2igvmd5";
     fetchSubmodules = true;
   };
 

